receiver/receivercreator/config_test.go

```diff
@@ -57,6 +57,14 @@ func (mh *mockHostFactories) GetExtensions() map[component.ID]component.Componen
 	return mh.extensions
 }
 
+var portRule = func(s string) rule {
```

Comment:
The linter required this change since `newRuleOrPanic()` was always used with the same argument.